recursive_art3.py

```diff
@@ -1,91 +1,80 @@
-""" This is my computational art project!
-    Anna Buchele """
+""" 
```

Comment:
Good job! Use git for version control so you don't need to make new files though.